guessit/config/options.json

```diff
@@ -734,6 +734,7 @@
       "ITV": "ITV",
       "Knowledge Network": "KNOW",
       "Lifetime": "LIFE",
+      "Max": "MAX",
```

Comment:
Shouldn't this be in "HBO Max" ?

Reply:
MAX and HBO MAX are now different streaming services.  Which one you get as a customer depends on region, but they have different encodes and selections.